Add Link to doxygen API ref (#259)
eProsima/Fast-DDS-docs: docs/fastdds/api_reference/api_reference.rst

API Reference
=============

*Fast DDS*, as a Data Distribution Service (DDS) standard implementation, exposes the DDS Data-Centric Publish-Subscribe
(DCPS) Platform Independent Model (PIM) API, as specified in the
`DDS specification <https://www.omg.org/spec/DDS/1.4/PDF>`_.
Furthermore, is also gives the user the possibility to directly interact with the underlying Real-time
Publish-Subscribe (RTPS) API that DDS implements for wired communications, as specified in the
`RTPS standard <https://www.omg.org/spec/DDSI-RTPS/2.2/PDF>`_.

This section presents the most commonly used APIs provided by *Fast DDS*.

> For a complete API reference, please refer to

Reviewer: As far as I see this API reference only includes the old Fast RTPS PubSub API. Consequently, I suggest saying:

```suggestion
For more information about the API reference, please refer to
```
